schemas/records.json

```diff
@@ -0,0 +1,775 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
```

Comment:
Had to change the JSON Schema version to draft-07 as 202012 is not supported by stac-node-validator right now, shouldn't change anything though.